.github/CODEOWNERS

```diff
@@ -1,4 +1,4 @@
-build/commands/ @bridiver
+build/commands/ @brave/build-commands-reviewers
```

Comment:
https://github.com/orgs/brave/teams/build-commands-reviewers
can someone look into this group? GitHub complains currently:

```
Unknown owner on line 1: make sure the team @brave/build-commands-reviewers exists, is publicly visible, and has write access to the repository
build/commands/ @brave/build-commands-reviewers
```

Reply:
Fixed.